diff --git a/src/execution/operator/persistent/physical_create_bf.cpp b/src/execution/operator/persistent/physical_create_bf.cpp
index 4b3362206a..f1e8d15d58 100644
--- a/src/execution/operator/persistent/physical_create_bf.cpp
+++ b/src/execution/operator/persistent/physical_create_bf.cpp
@@ -227,17 +227,17 @@ bool PhysicalCreateBF::GiveUpBFCreation(const DataChunk &chunk, OperatorSinkInpu
 			double row_length = static_cast<double>(gstate.total_row_size) / input_rows;
 			
 			// TODO: Currently, the number of threads affect the accuracy of progress percent.
-			if (gstate.num_threads > 8) {
-				if (selectivity > 0.35 || (row_length > 40 && selectivity > 0.2)) {
-					is_successful = false;
-					return true;
-				}
-			} else {
-				if (progress_percent < 0.65 && (selectivity > 0.35 || (row_length > 40 && selectivity > 0.2))) {
-					is_successful = false;
-					return true;
-				}
-			}
+			// if (gstate.num_threads > 8) {
+			// 	if (selectivity > 0.35 || (row_length > 40 && selectivity > 0.2)) {
+			// 		is_successful = false;
+			// 		return true;
+			// 	}
+			// } else {
+			// 	if (progress_percent < 0.65 && (selectivity > 0.35 || (row_length > 40 && selectivity > 0.2))) {
+			// 		is_successful = false;
+			// 		return true;
+			// 	}
+			// }
 
 			// 2. Estimate the lower bound of required memory, which is used to materialize this table. If it is very
 			// large, give up creating BF.
diff --git a/src/execution/transfer_bf_linker.cpp b/src/execution/transfer_bf_linker.cpp
index 89125e180f..979a0a32bf 100644
--- a/src/execution/transfer_bf_linker.cpp
+++ b/src/execution/transfer_bf_linker.cpp
@@ -19,11 +19,11 @@ void TransferBFLinker::LinkBFOperators(LogicalOperator &op) {
 	state = State::CLEAN_USELESS_OPERATORS;
 	VisitOperator(op);
 
-	state = State::UPDATE_MIN_MAX_BINDING;
-	VisitOperator(op);
-
-	state = State::SMOOTH_MARK_JOIN;
-	VisitOperator(op);
+	// state = State::UPDATE_MIN_MAX_BINDING;
+	// VisitOperator(op);
+	//
+	// state = State::SMOOTH_MARK_JOIN;
+	// VisitOperator(op);
 }
 
 void TransferBFLinker::VisitOperator(LogicalOperator &op) {
diff --git a/src/optimizer/predicate_transfer/transfer_graph_manager.cpp b/src/optimizer/predicate_transfer/transfer_graph_manager.cpp
index efd9030caa..9ba0108e39 100644
--- a/src/optimizer/predicate_transfer/transfer_graph_manager.cpp
+++ b/src/optimizer/predicate_transfer/transfer_graph_manager.cpp
@@ -61,10 +61,11 @@ bool TransferGraphManager::Build(LogicalOperator &plan) {
 	}
 
 	// 3. Unfiltered Table only receives Bloom filters, they will not generate Bloom filters.
-	SkipUnfilteredTable(joins);
+	// SkipUnfilteredTable(joins);
 
 	// 4. Create the transfer graph
-	CreateTransferPlanUpdated();
+	// CreateTransferPlanUpdated();
+	CreateOriginTransferPlan();
 
 	return true;
 }
